src/main/java/org/fcrepo/auth/webac/LinkHeaderProvider.java

```diff
@@ -31,6 +34,7 @@
 import org.fcrepo.kernel.modeshape.rdf.impl.DefaultIdentifierTranslator;
 import org.fcrepo.kernel.modeshape.rdf.impl.PropertiesRdfContext;
 
+import org.apache.commons.lang3.tuple.Pair;
```

Comment:
```Pair``` comes from the [```getEffectiveAcl```](https://github.com/fcrepo4/fcrepo-module-auth-webac/blob/master/src/main/java/org/fcrepo/auth/webac/WebACRolesProvider.java#L390). I'd rather not start messing with that function (if I can avoid it). Is there something I can use to consume a ```Pair``` what would be more palatable?